### DISCUSSION TITLE
Add ability to use a different email address for each team

### DISCUSSION THREAD

**Discussion by thinkswan** [vercel/community#1514] [Feature Question] [answered]
Hi! 👋 I use Vercel for both personal and work projects and would like notifications to go to a different email for each team. At the moment, all Vercel notifications go to my personal email, which is associated with my Vercel account.

Would it be possible to add a feature that allows you to set a different email address for each team you're a member of?

To describe what I'm seeking, GitHub offers this capability with a feature called [custom routing](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#customizing-email-routes-per-organization).

Thanks so much for considering this! 🙏

**Comment by AlbertMarashi** [comment]
+1 need this. Github supports multiple alternative emails, and having a vercel account should work the same way

**Comment by dizzyjaguar** [comment]
+1 as well!

**Comment by srossross** [comment]
this would be very useful

**Comment by guillermodlpa** [comment]
yes please :D

**Comment by rupert-br** [comment]
+1 as well!

**Comment by leerob** [comment]
We're working on the ability to support multiple email addresses under one account 🙏

**Reply by dizzyjaguar** [reply]
lets go!! great to hear

**Reply by GrzegorzDerdak** [reply]
@leerob Any ETA when we can expect it to be available?

**Comment by sglim** [comment]
+1 as well!

**Comment by MilesV64** [comment]
+1!! I don't even understand how to access my personal email Vercel account after linking my work email, so I can't even change plans or anything...

**Comment by leerob** [comment, marked as answer]
Good news everyone! This has now shipped.

https://twitter.com/vercel_changes/status/1785372987975209303

**Reply by tyler-dot-earth** [reply]
:point_up: This is not a solution to the problem in this thread. While you've added support for multiple email addresses for one account, it doesn't allow routing notifications on a per-project/per-org basis like GitHub does.

See how GitHub notifications work: https://github.com/settings/emails#preferences

This discussion should be re-opened.

**Reply by tyler-dot-earth** [reply]
I got an update from support:
> This isn't something we currently support, however it's definitely something that's on our roadmap to add. 
> 
> The multiple email address feature was only added recently and was the first step to being able to support this - now it's released we can build on the feature to add custom email routing and fallback options. Understandably this takes time and heavy testing, but it's definitely something we're aiming to add in the future.

So it sounds like it's on the way soon :tm:

**Comment by mjsilva** [comment]
sorry @leerob maybe I'm being dumb but even with the ability to add multiple emails to an account how can you route specific team notifications to a particular email? The new feature seems to just allow you to login with another email or I'm missing something?
